### PR TITLE
Run history release note change

### PR DESCRIPTION
**Pull request by mirnawong1**
Release note documenting run history data change submitted by Julia via [notion release note](https://www.notion.so/dbtlabs/dbt-Cloud-Run-History-Data-Retention-Changing-to-365-Days-c18ec23af1b647c3860d92ff3f33d373)

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: "Run history retention changing to 365 days"
+id: "run-history-endpoint"
+sidebar_label: "Update: Run history available for 365 days"
+sidebar_position: 6
+tags: [May-22-2023, API, Scheduler]
+---
+
+Beginning on June 1, 2023, dbt Cloud multi-tenant users can access run history for the previous 365 days through the dbt Cloud user interface (UI) and API, improving speed and accessibility.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
Beginning on June 1, 2023, dbt Cloud multi-tenant users can access run history for the previous 365 days through the dbt Cloud user interface (UI) and API. This update improves the application's speed and reliability, making it more performant for all users.
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: "Run history retention changing to 365 days"
+id: "run-history-endpoint"
+sidebar_label: "Update: Run history available for 365 days"
+sidebar_position: 6
+tags: [May-22-2023, API, Scheduler]
+---
+
+Beginning on June 1, 2023, dbt Cloud multi-tenant users can access run history for the previous 365 days through the dbt Cloud user interface (UI) and API. This update improves the application's speed and reliability, making it more performant for all users.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
Beginning on June 1, 2023, dbt Cloud multi-tenant users can access run history for the previous 365 days on a rolling basis through the dbt Cloud user interface (UI) and API. This update improves the application's speed and reliability, making it more performant for all users.
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+title: "Run history retention changing to 365 days"
+id: "run-history-endpoint"
+sidebar_label: "Update: Run history available for 365 days"
+sidebar_position: 6
+tags: [May-22-2023, API, Scheduler]
+---
+
+Beginning on June 1, 2023, dbt Cloud multi-tenant users can access run history for the previous 365 days on a rolling basis through the dbt Cloud user interface (UI) and API. This update improves the application's speed and reliability, making it more performant for all users.
+
+Specifically, all `GET` requests to the dbt Cloud [Runs endpoint](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs) will return information on runs, artifacts, logs, and run steps starting from June 1, 2022. The run history displayed in the dbt Cloud UI will also show runs from June 1, 2022 and onwards.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
Specifically, all `GET` requests to the dbt Cloud [Runs endpoint](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs) will return information on runs, artifacts, logs, and run steps only for the past 365 days.  The run history displayed in the dbt Cloud UI will only show runs for the past 365 days.  
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+tags: [May-22-2023, API, Scheduler]
+---
+
+Beginning on June 1, 2023, dbt Cloud multi-tenant users can access run history for the previous 365 days on a rolling basis through the dbt Cloud user interface (UI) and API. This update improves the application's speed and reliability, making it more performant for all users.
+
+Specifically, all `GET` requests to the dbt Cloud [Runs endpoint](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs) will return information on runs, artifacts, logs, and run steps only for the past 365 days.  The run history displayed in the dbt Cloud UI will only show runs for the past 365 days.  
+
+<Lightbox src="/img/docs/dbt-cloud/rn-run-history.jpg" width="100%" title="The dbt Cloud UI displaying a Run history"/>
+
+dbt Labs keeps older run events. To access run history older than 365 days, you'll need to contact dbt Cloud Support to request this. You can contact the Support team at [support@getdbt.com](mailto:support@getdbt.com) or using the dbt Cloud application chat for help by clicking the `?` icon in the dbt Cloud UI.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
dbt Labs will retain older run history until September 1, 2023, and will make it available to users who reach out to the Support team. To request older run history info, contact dbt Cloud Support to request at [support@getdbt.com](mailto:support@getdbt.com) or use the dbt Cloud application chat for help by clicking the `?` icon in the dbt Cloud UI. 
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+title: "Run history retention changing to 365 days"
+id: "run-history-endpoint"
+sidebar_label: "Update: Run history available for 365 days"
+sidebar_position: 6
+tags: [May-22-2023, API, Scheduler]
+---
+
+Beginning on June 1, 2023, dbt Cloud multi-tenant users can access run history for the previous 365 days on a rolling basis through the dbt Cloud user interface (UI) and API. This update improves the application's speed and reliability, making it more performant for all users.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
We are making a change to our data retention policy for run history in dbt Cloud.

Beginning on June 1, 2023, dbt Cloud multi-tenant users can access run history for the previous 365 days on a rolling basis through the dbt Cloud user interface (UI) and API. This update improves the application's speed and reliability, making it more performant for all users.
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+title: "Run history retention changing to 365 days"
+id: "run-history-endpoint"
+sidebar_label: "Update: Run history available for 365 days"
+sidebar_position: 6
+tags: [May-22-2023, API, Scheduler]
+---
+
+We are making a change to our data retention policy for run history in dbt Cloud.
+
+Beginning on June 1, 2023, dbt Cloud multi-tenant users can access run history for the previous 365 days on a rolling basis through the dbt Cloud user interface (UI) and API. This update improves the application's speed and reliability, making it more performant for all users.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion

**Beginning June 1, 2023**, developers on the dbt Cloud multi-tenant application will be able to self-serve access to their account’s run history through dbt Cloud user interface (UI) and API for only the precious 365 days, on a rolling basis. This update minimizes the amount of metadata we retain and while maximizing application performance for all users. 
```

**Review by mirnawong1 — COMMENTED** [review]
Updating language

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+sidebar_position: 6
+tags: [May-22-2023, API, Scheduler]
+---
+
+We are making a change to our data retention policy for run history in dbt Cloud.
+
+
+**Beginning June 1, 2023**, developers on the dbt Cloud multi-tenant application will be able to self-serve access to their account’s run history through dbt Cloud user interface (UI) and API for only the precious 365 days, on a rolling basis. This update minimizes the amount of metadata we retain and while maximizing application performance for all users. 
+
+Specifically, all `GET` requests to the dbt Cloud [Runs endpoint](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs) will return information on runs, artifacts, logs, and run steps only for the past 365 days.  The run history displayed in the dbt Cloud UI will only show runs for the past 365 days.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
Specifically, all `GET` requests to the dbt Cloud [Runs endpoint](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs) will return information on runs, artifacts, logs, and run steps only for the past 365 days.  Additionally, the run history displayed in the dbt Cloud UI will only show runs for the past 365 days.  
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+Specifically, all `GET` requests to the dbt Cloud [Runs endpoint](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs) will return information on runs, artifacts, logs, and run steps only for the past 365 days.  Additionally, the run history displayed in the dbt Cloud UI will only show runs for the past 365 days.  
+
+<Lightbox src="/img/docs/dbt-cloud/rn-run-history.jpg" width="100%" title="The dbt Cloud UI displaying a Run history"/>
+
+dbt Labs will retain older run history until September 1, 2023, and will make it available to users who reach out to the Support team. To request older run history info, contact dbt Cloud Support to request at [support@getdbt.com](mailto:support@getdbt.com) or use the dbt Cloud application chat for help by clicking the `?` icon in the dbt Cloud UI. 
+
+This is a one-time event on limiting the self-service of run history. dbt Labs may introduce a metadata retention policy in the future.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
+---
+title: "Run history retention changing to 365 days"
+id: "run-history-endpoint"
+sidebar_label: "Update: Run history available for 365 days"
+sidebar_position: 6
+tags: [May-22-2023, API, Scheduler]
+---
+
+We are making a change to our data retention policy for run history in dbt Cloud.
```
Comment (schottj) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
We are making a change to our metadata retrieval policy for run history in dbt Cloud. 
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+id: "run-history-endpoint"
+sidebar_label: "Update: Run history available for 365 days"
+sidebar_position: 6
+tags: [May-22-2023, API, Scheduler]
+---
+
+We are making a change to our metadata retrieval policy for run history in dbt Cloud. 
+
+
+**Beginning June 1, 2023**, developers on the dbt Cloud multi-tenant application will be able to self-serve access to their account’s run history through dbt Cloud user interface (UI) and API for only the precious 365 days, on a rolling basis. This update minimizes the amount of metadata we retain and while maximizing application performance for all users.
```
Comment (schottj) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
**Beginning June 1, 2023**, developers on the dbt Cloud multi-tenant application will be able to self-serve access to their account’s run history through dbt Cloud user interface (UI) and API for only 365 days, on a rolling basis. Older run history will be available for download by reaching out to Customer Support. We're seeking to minimize the amount of metadata we store while maximizing application performance. 

```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+
+
+**Beginning June 1, 2023**, developers on the dbt Cloud multi-tenant application will be able to self-serve access to their account’s run history through dbt Cloud user interface (UI) and API for only the precious 365 days, on a rolling basis. This update minimizes the amount of metadata we retain and while maximizing application performance for all users. 
+
+Specifically, all `GET` requests to the dbt Cloud [Runs endpoint](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs) will return information on runs, artifacts, logs, and run steps only for the past 365 days.  Additionally, the run history displayed in the dbt Cloud UI will only show runs for the past 365 days.  
+
+<Lightbox src="/img/docs/dbt-cloud/rn-run-history.jpg" width="100%" title="The dbt Cloud UI displaying a Run history"/>
+
+dbt Labs will retain older run history until September 1, 2023, and will make it available to users who reach out to the Support team. To request older run history info, contact dbt Cloud Support to request at [support@getdbt.com](mailto:support@getdbt.com) or use the dbt Cloud application chat for help by clicking the `?` icon in the dbt Cloud UI.
```
Comment (schottj) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
dbt Labs will retain older run history in cold storage and can make it available to customers who reach out to Customer Support. To request older run history info, contact dbt Cloud Support to request at [support@getdbt.com](mailto:support@getdbt.com) or use the dbt Cloud application chat for help by clicking the `?` icon in the dbt Cloud UI. 
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 12]
```diff
@@ -0,0 +1,19 @@
+---
+title: "Run history retention changing to 365 days"
```
Comment (schottj) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
title: "Run History Old Metadata Retrieval Change"
```

**Review by schottj — DISMISSED** [review]

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Run History Old Metadata Retrieval Change"
+id: "run-history-endpoint"
+sidebar_label: "Update: Run history available for 365 days"
```
Comment (schottj) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
sidebar_label: "Update: Old Run history retrieval change"
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Run History Old Metadata Retrieval Change"
```
Comment (schottj) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
title: "Older Run History Retrieval Change"
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+id: "run-history-endpoint"
+sidebar_label: "Update: Old Run history retrieval change"
```
Comment (schottj) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
sidebar_label: "Update: Older Run history retrieval change"
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 16]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Older Run History Retrieval Change"
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]: changing to sentence case per style guide

```suggestion
title: "Older run history retrieval change"
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 17]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Older run history retrieval change"
+id: "run-history-endpoint"
+sidebar_label: "Update: Older Run history retrieval change"
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]: changing to sentence case per style guide

```suggestion
sidebar_label: "Update: Older run history retrieval change"
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 18]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Older run history retrieval change"
+id: "run-history-endpoint"
+sidebar_label: "Update: Older run history retrieval change"
+sidebar_position: 6
+tags: [May-22-2023, API, Scheduler]
+---
+
+We are making a change to our metadata retrieval policy for run history in dbt Cloud.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
dbt Labs are making a change to our metadata retrieval policy for run history in dbt Cloud. 
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 19]
```diff
@@ -0,0 +1,20 @@
...
+
+
+Specifically, all `GET` requests to the dbt Cloud [Runs endpoint](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs) will return information on runs, artifacts, logs, and run steps only for the past 365 days.  Additionally, the run history displayed in the dbt Cloud UI will only show runs for the past 365 days.  
+
+<Lightbox src="/img/docs/dbt-cloud/rn-run-history.jpg" width="100%" title="The dbt Cloud UI displaying a Run history"/>
+
+dbt Labs will retain older run history in cold storage and can make it available to customers who reach out to Customer Support. To request older run history info, contact dbt Cloud Support to request at [support@getdbt.com](mailto:support@getdbt.com) or use the dbt Cloud application chat for help by clicking the `?` icon in the dbt Cloud UI.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
We will retain older run history in cold storage and can make it available to customers who reach out to our Support team. To request older run history info, contact the Support team at [support@getdbt.com](mailto:support@getdbt.com) or use the dbt Cloud application chat for help by clicking the `?` icon in the dbt Cloud UI. 
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 20]
```diff
@@ -0,0 +1,20 @@
...
+id: "run-history-endpoint"
+sidebar_label: "Update: Older run history retrieval change"
+sidebar_position: 6
+tags: [May-22-2023, API, Scheduler]
+---
+
+dbt Labs are making a change to our metadata retrieval policy for run history in dbt Cloud. 
+
+
+**Beginning June 1, 2023**, developers on the dbt Cloud multi-tenant application will be able to self-serve access to their account’s run history through dbt Cloud user interface (UI) and API for only 365 days, on a rolling basis. Older run history will be available for download by reaching out to Customer Support. We're seeking to minimize the amount of metadata we store while maximizing application performance.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
**Beginning June 1, 2023,** developers on the dbt Cloud multi-tenant application will be able to self-serve access to their account’s run history through the dbt Cloud user interface (UI) and API for only 365 days, on a rolling basis. Older run history will be available for download by reaching out to Customer Support. We're seeking to minimize the amount of metadata we store while maximizing application performance. 
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 21]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Older run history retrieval change"
+id: "run-history-endpoint"
+sidebar_label: "Update: Older run history retrieval change"
+sidebar_position: 6
+tags: [May-22-2023, API, Scheduler]
+---
+
+dbt Labs are making a change to our metadata retrieval policy for run history in dbt Cloud.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
dbt Labs is making a change to the metadata retrieval policy for run history in dbt Cloud. 
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 22]
```diff
@@ -0,0 +1,20 @@
...
+
+
+**Beginning June 1, 2023,** developers on the dbt Cloud multi-tenant application will be able to self-serve access to their account’s run history through the dbt Cloud user interface (UI) and API for only 365 days, on a rolling basis. Older run history will be available for download by reaching out to Customer Support. We're seeking to minimize the amount of metadata we store while maximizing application performance. 
+
+
+Specifically, all `GET` requests to the dbt Cloud [Runs endpoint](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs) will return information on runs, artifacts, logs, and run steps only for the past 365 days.  Additionally, the run history displayed in the dbt Cloud UI will only show runs for the past 365 days.  
+
+<Lightbox src="/img/docs/dbt-cloud/rn-run-history.jpg" width="100%" title="The dbt Cloud UI displaying a Run history"/>
+
+We will retain older run history in cold storage and can make it available to customers who reach out to our Support team. To request older run history info, contact the Support team at [support@getdbt.com](mailto:support@getdbt.com) or use the dbt Cloud application chat for help by clicking the `?` icon in the dbt Cloud UI.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
We will retain older run history in cold storage and can make it available to customers who reach out to our Support team. To request older run history info, contact the Support team at [support@getdbt.com](mailto:support@getdbt.com) or use the dbt Cloud application chat by clicking the `?` icon in the dbt Cloud UI. 
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 23]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Older run history retrieval change"
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
title: "Older Run History retrieval change"
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 24]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Older Run History retrieval change"
+id: "run-history-endpoint"
+sidebar_label: "Update: Older run history retrieval change"
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
sidebar_label: "Update: Older Run History retrieval change"
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 25]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Older Run History retrieval change"
+id: "run-history-endpoint"
+sidebar_label: "Update: Older Run History retrieval change"
+sidebar_position: 6
+tags: [May-22-2023, API, Scheduler]
+---
+
+dbt Labs is making a change to the metadata retrieval policy for run history in dbt Cloud.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
dbt Labs is making a change to the metadata retrieval policy for Run History in dbt Cloud. 
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md [hunk 26]
```diff
@@ -0,0 +1,20 @@
...
+
+dbt Labs is making a change to the metadata retrieval policy for Run History in dbt Cloud. 
+
+
+**Beginning June 1, 2023,** developers on the dbt Cloud multi-tenant application will be able to self-serve access to their account’s run history through the dbt Cloud user interface (UI) and API for only 365 days, on a rolling basis. Older run history will be available for download by reaching out to Customer Support. We're seeking to minimize the amount of metadata we store while maximizing application performance. 
+
+
+Specifically, all `GET` requests to the dbt Cloud [Runs endpoint](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs) will return information on runs, artifacts, logs, and run steps only for the past 365 days.  Additionally, the run history displayed in the dbt Cloud UI will only show runs for the past 365 days.  
+
+<Lightbox src="/img/docs/dbt-cloud/rn-run-history.jpg" width="100%" title="The dbt Cloud UI displaying a Run history"/>
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md]:
```suggestion
<Lightbox src="/img/docs/dbt-cloud/rn-run-history.jpg" width="100%" title="The dbt Cloud UI displaying a Run History"/>
```

**Comment by mirnawong1** [comment]
Julia approved this already before some changes were made, and has approved this pr to merge. merging this pr now

**Pull request merged**